### DISCUSSION TITLE
Reopen PR after closing it

### DISCUSSION THREAD

**Discussion by arunsathiya** [community/community#88753] [Mobile] [answered]
### Select Topic Area

Question

### Body

I see there's an option to close PRs from the iOS app (haven't checked Android) but there's one to reopen the PR after that. I had to do that from the website. Would be nice to have that feature too.

**Comment by pamelaacme** [comment]
> ### Select Topic Area
> 
> Question
> 
> ### Body
> 
> I see there's an option to close PRs from the iOS app (haven't checked Android) but there's one to reopen the PR after that. I had to do that from the website. Would be nice to have that feature too.

**Comment by mishmanners** [comment, marked as answer]
yes you can close a PR from GitHub Mobile:
1. Navigate to "Pull Requests" from the home screen
2. Click the second drop down and select "Closed"
3. Select a pull request that has been closed and click on it (note, it has to be a closed PR, not a PR that's closed because it's been merged)
4. Click the three dots at the top of the screen (the options menu)
5. Click "reopen Pull Request"

You can also close a PR in the same way.

**Comment by MichaelChirico** [comment]
Note that this DOES NOT WORK if the closed PR _targets a branch which is deleted_.

I often have PR chains like `main` < `A` < `B` < ... < `N`.

If branch `A` is deleted and PR `B` is closed, it will be impossible to re-open `B` without first restoring branch `A`. I don't know why, it feels like a bug.

If instead `B` remains open when `A` is merged and its branch deleted, GitHub automatically sets the target to `main`, which is the desired behavior, however we can't edit the target branch of a closed PR.

**Reply by glevco** [reply]
Yes, this was the behavior and it regressed recently, so I also think it's a bug. @mishmanners
